docker-scripts/klaw-docker.sh

```diff
@@ -21,6 +20,19 @@ build () {
 
 }
 
+# This is only for use in E2E tests. It skips testing to
+# speed up the process. This assumes that the code it runs
+# against is unit/integration tested.
```

Comment:
Would you mind adding to the comment that it is not included in the usage because it is intended for advance pipeline usage only?

Reply:
Good point! 

https://github.com/Aiven-Open/klaw/pull/1738/commits/0aa5f4f6b435047288af84d3de223f386c346bb9